utils/iotop/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2017 Bjørn Mork <bjorn@mork.no>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=iotop
+PKG_VERSION:=0.6
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Bjørn Mork <bjorn@mork.no>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://guichaz.free.fr/iotop/files/
```

Comment:
Won't the default "sources" mirrors be sufficient for any such short availability glitches?  I thought about using the git repo at repo.or.cz as a secondary source, but couldn't figure out how to both a version and a tag based download.  Which is probably a feature.

Switching to the git repo is of course also an option.  But the named website *is* the real upstream release source, and I did want to base the package on released versions.